__tests__/api.test.js

```diff
@@ -38,3 +38,30 @@ describe("GET: /api/topics", () => {
       });
   });
 });
+
+describe("GET: /api", () => {
+  test("200: should return a full list of objects from the endpoints.json file", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        const endpointsObj = res.body.endpoints;
+        const endpointsArr = Object.values(endpointsObj);
+        endpointsArr.forEach((endpoint) => {
+          expect(endpoint).toMatchObject({
+            description: expect.any(String),
+            queries: expect.any(Array),
+            exampleResponse: expect.any(Object),
+          });
+        });
+      });
+  });
+  test("404: ERROR - responds with the path is not found", () => {
```

Comment:
This is a good test but deals with any invalid endpoint so shouldn't be in the Get /api description box